pkgs/top-level/python-packages.nix

```diff
@@ -1608,6 +1608,11 @@ in {
 
   dugong = callPackage ../development/python-modules/dugong {};
 
+  easysnmp = callPackage ../development/python-modules/easysnmp {
+    openssl = pkgs.openssl;
```

Comment:
This shouldn't be necessary, pythonPackages will fill missing stuff in from pkgs.

Reply:
i recommended this, in case there's a package added to python-modules of that name. I feel like it's worth-while future-proofing

Reply:
Makes sense to me

Reply:
Still :-1: on this. I'd be against adding something called `openssl` or `net_snmp` to python-packages anyway, and there are plenty of other packages using non-python deps without them being threaded through specifically. (also, if you really want to, you can rephrase this as `inherit (pkgs) openssl net_snmp`).

Reply:
I do see lots of examples of the `inherit (pkgs)` pattern. Shall we use it here?

Reply:
I guess it's just because I'm not a big fan of bringing in a entire package sets. I wonder what @FRidh  has to say on pkgs vs per-package imports

Reply:
Rebased on master again. I'd like to do whatever we need to do to get this merged. Sounds like there's disagreement on which way to go, so we need an authoritative decision.

Reply:
Pinging @jonringer, @lheckemann, and @FRidh. I need to know which way to go with this change. Please advise.

Reply:
I'm still in favor of the current state. I'm not a committer, so I can't merge :(

Reply:
@jonringer, do you know who else we might try pinging?